### PR TITLE
GHP: Rest API v3 desc

### LINKED ISSUE #2092: Document the new REST API of Enceladus 3
## Background
There's a new REST API in Enceladus 3

## Feature
Add a description of reference to the REST API into https://absaoss.github.io/enceladus/

### PR DESCRIPTION
**Pull request by dk1844**
- Enceladus v3 REST API info has been added to GH pages (basic info, linking to built-in Swagger mini-site). 
 - V2-like Menas API doc page has been removed.

Closes #2092

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -25,10 +25,12 @@ GEM
     ethon (0.12.0)
       ffi (>= 1.3.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```
Comment (Zejnilovic) [inline Gemfile.lock:28]: This is weird. Why are there 2 eventmachine dependencies?

**Review by Zejnilovic — APPROVED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (Zejnilovic) [reply Gemfile.lock:28]: So this would be windows one? It should be fine, but if the build/site breaks, I would look at this as the first culprit.

Comment (benedeki) [reply Gemfile.lock:28]: I have Gemfile.lock exempt from commit, I believe

**Review by benedeki — APPROVED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (dk1844) [reply Gemfile.lock:28]: Hmm, maybe we should exempt it here, too, later on.

**Pull request merged**